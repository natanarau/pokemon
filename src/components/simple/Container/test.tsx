import { render, waitFor, cleanup } from '@testing-library/react';
import { Container } from '.';


afterEach(() => {
  cleanup();
  jest.resetModules();
});

describe('<Container />', () => {
  test('should render icon correctly', async () => {
    const { getByTestId } = render(<Container>Container</Container>)
    await waitFor(() => expect(getByTestId('Container')).toBeTruthy())
  })
  test('should match snapshot', () => {
    const { container } = render(
      <Container>Container</Container>
    )
    expect(container).toMatchSnapshot();
  });
})