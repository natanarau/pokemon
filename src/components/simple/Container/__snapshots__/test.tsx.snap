// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should match snapshot 1`] = `
.c0 {
  max-width: container.xl;
  margin: auto;
}

<div>
  <div
    class="c0"
    data-testid="Container"
  >
    Container
  </div>
</div>
`;
